### PR TITLE
Major refactor of Fluo docs

### PR DESCRIPTION
**Pull request by mikewalch**
* Added docker and yarn instructions
* Split up applications page
* Create quick start

### CODE REVIEW

File: _fluo-1-2/administration/run-fluo-in-docker.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+| [Fluo]      | 1.2.0   |
+| [Accumulo]  | 1.8.1   |
+| [Hadoop]    | 2.7.3   |
+| [Zookeeper] | 3.4.9   |
+
+If these versions do not match what is running on your cluster, you should consider building
+your own image with matching versions. However, Fluo must be 1.2+. Below are instructions for
+building an image:
+
+1. Clone the Fluo docker repo
```
Comment (keith-turner) [inline _fluo-1-2/administration/run-fluo-in-docker.md]: We could put the build instructions in the projects readme and reference them from here.   This document would only have the instructions for running.

Comment (mikewalch) [reply _fluo-1-2/administration/run-fluo-in-docker.md]: Good idea. I will do that.

File: _fluo-1-2/administration/initialize.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+5. Run `fluo config myapp` to see what configuration is stored in ZooKeeper.
+
+## Next Steps
+
+Run your Fluo application using one of the methods below:
+
+* [Run Fluo processes][process]
+* [Run Fluo in YARN][yarn]
+* [Run Fluo in Docker][docker]
```
Comment (keith-turner) [inline _fluo-1-2/administration/initialize.md]: Would it be worthwhile to mention Mesos and Kubernetes here?  Something like `[Run Fluo in Docker][docker] which enables running in Mesos and Kubernetes`

File: _fluo-1-2/administration/run-fluo-in-docker.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+Next, choose a method below to run the oracle and worker(s) of your Fluo application. In the examples below, the Fluo
+application is named `myapp` and was initialized using a Zookeeper node on `zkhost`.
+
+### Docker engine
+
+Use the `docker` command to start local docker containers.
+
+1. Start a Fluo oracle
+
+        docker run -d --network="host" apache/fluo oracle -a myapp -o fluo.connection.zookeepers=zkhost/fluo
```
Comment (keith-turner) [inline _fluo-1-2/administration/run-fluo-in-docker.md]: could give a one sentence description of what each argument and option does.

**Review by keith-turner — APPROVED** [review]

File: _fluo-1-2/administration/initialize.md (hunk 2 above)
Comment (mikewalch) [reply _fluo-1-2/administration/initialize.md]: Fixed

File: _fluo-1-2/administration/run-fluo-in-docker.md (hunk 3 above)
Comment (mikewalch) [reply _fluo-1-2/administration/run-fluo-in-docker.md]: Fixed

**Pull request merged**